README.md

```diff
@@ -107,7 +107,16 @@ Follow the steps below to set up a testnet smart wallet using `starkli`:
 
 ### Ethereum smart contract
 
-First, the Ethereum smart contracts must be deployed. For Ethereum the deployment process you will need to:
+First, make sure you have the necessary dependencies installed in the `contracts/solidity/lib` folder.
+You can install these commands by executing them within the `/contracts/solidity/` folder.
+
+- **[openzeppelin-contracts-upgradeable]**(https://github.com/OpenZeppelin/openzeppelin-foundry-upgrades)
+`forge install OpenZeppelin/openzeppelin-contracts-upgradeable`
+
+- **[forge-std]**(https://github.com/foundry-rs/forge-std)
+`forge install foundry-rs/forge-std `
```

Comment:
```suggestion
- [openzeppelin-contracts-upgradeable](https://github.com/OpenZeppelin/openzeppelin-foundry-upgrades)
`forge install OpenZeppelin/openzeppelin-contracts-upgradeable`

- [forge-std](https://github.com/foundry-rs/forge-std)
`forge install foundry-rs/forge-std `
```
i think this .md links are not rendering correctly

Reply:
I would also separate in different lines the link from the command

Reply:
+1